pkg/trace/obfuscate/sql_tokenizer.go

```diff
@@ -223,6 +223,9 @@ func (tkn *SQLTokenizer) Scan() (TokenKind, []byte) {
 				tkn.advance()
 				return ColonCast, []byte("::")
 			}
+			if unicode.IsSpace(tkn.lastChar) {
+				return TokenKind(ch), tkn.bytes()
```

Comment:
```suggestion
				// example scenario: "autovacuum: VACUUM ANALYZE fake.table"
				return TokenKind(ch), tkn.bytes()
```
Just to be clear here. This code is fairly complex so we have to make sure it can be figured out without excessive digging around git blaming and PRs. In this case it's hard because we don't have a specific TokenKind, so the comment helps.